.dockerignore

```diff
@@ -0,0 +1,2 @@
+.git
```

Comment:
Is this file added intentionally?

Reply:
ugh woops I had that in a separate PR and checked it in accidentally because it was slowing down building docker containers a lot on my local. I'll get rid of that